docs/site/content/posts/02-vmware-tanzu-community-edition-takes-off.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "VMware Tanzu Community Edition Takes Off!"
+slug: tanzu-community-edition-takes-off
+date: 2021-11-11
+author: Jennifer Kotzen
+image: /img/rocket.jpg
+excerpt: "“Pleased” is a nice word, and “delighted” is too, but neither quite captures the way we’re feeling right now. “Excited” is better; “elated” is getting close ... “ecstatic” maybe? You get the idea. The simple truth is that we are beyond thrilled to see the tremendous response to the release of VMware Tanzu Community Edition! In case you missed the excitement around the introduction of this freely available Kubernetes platform for learners and users, allow me to fill you in."
+tags: ['Jennifer Kotzen']
+---
+![](/img/rocket.jpg)
+
+"Pleased" is a nice word, and "delighted" is too, but neither quite
+captures the way we're feeling right now. "Excited" is better; "elated"
+is getting close \... "ecstatic" maybe? You get the idea. The simple
+truth is that we are beyond thrilled to see the tremendous response to
+the release of VMware Tanzu Community Edition! In case you missed the
+excitement around the introduction of this freely available Kubernetes
+platform for learners and users, allow me to fill you in.
+
+Just a few weeks ago, on October 4, [Tanzu Community
+Edition](http://www.tanzucommunityedition.io) was released.
+Immediate uptake has been explosive and amazing, and it continues to
+expand at an incredible rate. To date we've seen hundreds of thousands
+of web mentions and tens of thousands of visitors to our web site.
+Downloads are off the charts, and thousands of you have already begun
+exploring Tanzu Community Edition through hands-on workshops. Wow! What
+a great start for our community. 😊
+
+If you are new to the party, you can quickly catch up on Tanzu Community
+Edition release news and perspectives from a few of our favorite blog
+posts:
+
+-   In VMware's [formal announcement](https://tanzu.vmware.com/content/blog/vmware-tanzu-community-edition-announcement) of Tanzu Community Edition, Amanda Katona breaks the news about the release. Take a quick read to catch up on what Tanzu Community Edition is, who it is for, and how it can be used.
+
+-   Nigel Brown and Roger Klorese [welcome all to collaborate and contribute](https://tanzucommunityedition.io/posts/community-is-our-middle-name-welcome/) together in the community's inaugural blog.
+
+-   Always insightful, [Craig McLuckie's blog post](https://tanzu.vmware.com/content/blog/vmware-tanzu-community-edition-kubernetes-cloud-native-ecosystem) discusses the importance of open source technologies in the cloud
+native landscape and the value of being able to easily consume them with Tanzu Community Edition.
+
+-   Especially if you're a developer, you'll want to check out [Grant
+ Shipley's suggestions](https://tanzu.vmware.com/developer/blog/expanding-the-developer-toolbox-with-vmware-tanzu-community-edition/) for a few open source tools that you'll want to use with Tanzu Community Edition.
```

Comment:
Here too. Need to remove extra spaces after the opening `-`, then align this with the start of the text there.

Reply:
One more space needed to line up.

Reply:
Same here with the `S` and `E`.